src/plugins/unified_search/public/filter_bar/filter_editor/lib/filter_editor_utils.ts

```diff
@@ -63,7 +63,7 @@ export function isFilterValid(
   operator?: Operator,
   params?: any
 ) {
-  if (!indexPattern || !field || !operator) {
+  if (!indexPattern || !field || !operator || !params) {
```

Comment:
@nlatipov please doube check that change.  I just see that for 'exist' filter you did 

<img width="607" alt="image" src="https://user-images.githubusercontent.com/20072247/199110296-b0c25d2a-877e-44ba-8576-eb9a3f395ad4.png">

Reply:
I've temporary removed that, if we still need that for some reason, let's discuss it offline.

Reply:
Thanks for bringing this matter to your attention.
I added an additional check 

```
  if (operator.type !== FILTERS.EXISTS && !params) {
    return false;
  }
```

Now if the operator is existential the check will be correct. And also the check will remain correct for other cases